contracts/governnace/GovernorOHMegaInterfaces.sol

```diff
@@ -0,0 +1,194 @@
+
+        /// @notice the ordered list of target addresses for calls to be made
+        address[] targets;
+
+        /// @notice The ordered list of values (i.e. msg.value) to be passed to the calls to be made
+        uint[] values;
+
+        /// @notice The ordered list of function signatures to be called
+        string[] signatures;
+
+        /// @notice The ordered list of calldata to be passed to each call
+        bytes[] calldatas;
+
+        /// @notice The block at which voting begins: holders must delegate their votes prior to this block
+        uint startBlock;
+
+        /// @notice The block at which voting ends: votes must be cast prior to this block
+        uint endBlock;
+
+        /// @notice Current number of votes in favor of this proposal
+        uint forVotes;
+
+        /// @notice Current number of votes in opposition to this proposal
+        uint againstVotes;
+
+        /// @notice Current number of votes for abstaining for this proposal
+        uint abstainVotes;
+
+        /// @notice Threshold of wsOHM at start of proposal
+        /// @notice change from original contract
+        uint thresholdAtStart;
+
+        /// @notice Number of wsOHM needed to pass vote
+        /// @notice change from original contract
+        uint votesNeeded;
+
+        /// @notice Flag marking whether the proposal has been canceled
+        bool canceled;
+
+        /// @notice Flag marking whether the proposal has been executed
+        bool executed;
+
+        /// @notice Receipts of ballots for the entire set of voters
+        mapping (address => Receipt) receipts;
+    }
+
+    /// @notice Ballot receipt record for a voter
+    struct Receipt {
+        /// @notice Whether or not a vote has been cast
+        bool hasVoted;
+
+        /// @notice Whether or not the voter supports the proposal or abstains
+        uint8 support;
+
+        /// @notice The number of votes the voter had, which were cast
+        uint votes;
+    }
+
+    /// @notice Possible states that a proposal may be in
+    enum ProposalState {
+        Pending,
+        Active,
+        Canceled,
+        Defeated,
+        Succeeded,
+        Queued,
+        Expired,
+        Executed
+    }
+}
+
+interface TimelockInterface {
+    function delay() external view returns (uint);
+    function GRACE_PERIOD() external view returns (uint);
+    function acceptAdmin() external;
+    function queuedTransactions(bytes32 hash) external view returns (bool);
+    function queueTransaction(address target, uint value, string calldata signature, bytes calldata data, uint eta) external returns (bytes32);
+    function cancelTransaction(address target, uint value, string calldata signature, bytes calldata data, uint eta) external;
+    function executeTransaction(address target, uint value, string calldata signature, bytes calldata data, uint eta) external payable returns (bytes memory);
+}
+
+/// @notice change from original contract
+interface wsOHMInterface {
```

Comment:
IMO this would be worth putting in its own file and include all of the special functions (wrapping, unwrapping, etc). wsOHM will be used in other contracts as well, so I think having a single file with its interface would be helpful for people trying to build on top of Olympus.

Reply:
Agreed Good call!. we're currently unflattening the rest of the repo and we can pull out utility methods appropriately as well.